src/site/topic26.rst

```diff
@@ -119,6 +119,61 @@ Recursively Thinking
         * Return the sorted merged list
 
 
+Complexity Analysis
+-------------------
+
+* A simple way to think about the analysis is to consider
+    * How much work is involved for a single merging of two lists
+    * How many times merging needs to happen
+
+* It can get more nuanced, but this level of detail is sufficient
+
+
+Merging
+^^^^^^^
+
+* We will have two lists to merge into one
+* One list will be size :math:`n`, the other will be size :math:`m`
+* These lists will either be the same length or nearly the same length, but either way, we can say :math:`n >= m`
+* Since we're concerned about worst case scenario, we will consider the case where :math:`n = m`, therefore we have :math:`2n` elements total
+* There will be some number of comparisons taking place, but ultimately, each of these :math:`2n` elements will need to be moved to the merged list
+* Therefore, because we ignore coefficients, the complexity of merging is :math:`O(n)`
+
+
+Number of Merges
+^^^^^^^^^^^^^^^^
```

Comment:
Yeah, this definitely do be being chonky. They should be fairly familiar with O(logn) by now, you can also relate to binary search which splits the search space in half each step to show it takes logn divisions to get to a single element. Then, at each of those divisions, you do merge. So O(n) * O(logn).